server/cp/macros/update_translation_metadata_macro.py

```diff
@@ -17,13 +17,20 @@
 
 def update_translation_metadata_macro(item, **kwargs):
     if item.get('anpa_take_key'):
-        item['anpa_take_key'] = '0'
+        item['anpa_take_key'] = ''
+
+    if item.get('correction_sequence'):
+        item['correction_sequence'] = 0
 
     cv = get_resource_service('vocabularies').find_one(req=None, _id='destinations')
     if not cv or not cv.get('items'):
         return
+
     for value in cv['items']:
-        if value.get('name') == 'Presse Canadienne staff':
+        subject = item.get('subject', [])
+        is_destination = any(sub for sub in subject if sub.get('name') == 'Presse Canadienne staff')
```

Comment:
seems to me like you don't have to do this for every value in cv but rather before and then user break when you add it